### PR TITLE
Esri debug

### PR DESCRIPTION
**Pull request by kylemann16**
Changed from OBB calculations to Cube and moved srs to initialize and output messages about the data set to ready. Ticket #2334

### CODE REVIEW

File: plugins/i3s/io/EsriReader.cpp [hunk 1]
```diff
@@ -144,21 +145,31 @@ void EsriReader::initialize(PointTableRef table)
...
-    log()->get(LogLevel::Debug) << "min_density: " <<
-        m_args.min_density << std::endl;
-    log()->get(LogLevel::Debug) << "max_density: " <<
-        m_args.max_density << std::endl;
-    log()->get(LogLevel::Debug) << "dimensions: " << std::endl;
-    for (std::string& dim : m_args.dimensions)
+    //create spatial reference objects
+    Json::Value spatialJson = m_info["spatialReference"];
+    std::string spatialStr = "EPSG:" + spatialJson["wkid"].asString();
+    std::cout << "SRS: " << spatialStr << std::endl;
```
Comment (connormanning) [inline plugins/i3s/io/EsriReader.cpp]: Use logger.

File: test/data/i3s/plineSonoma3.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (connormanning) [inline test/data/i3s/plineSonoma3.json]: This file is added but tests were not changed - is this used?

Comment (kylemann16) [reply test/data/i3s/plineSonoma3.json]: It was something weird... I wasn't aware I changed it, but I changed it back to where it was before.

File: plugins/i3s/io/EsriReader.cpp (hunk 1 above)
Comment (kylemann16) [reply plugins/i3s/io/EsriReader.cpp]: Just got rid of it.

**Comment by abellgithub** [comment]
Is this something we really want in master?  Don't we want to get feedback from ESRI to figure out if this is what we actually want first?

**Comment by hobu** [comment]
We never heard back from ESRI on this topic, but this PR does fix the issue in #2334. Merge for now for 1.9 and if something comes up we can address that in the next release.

**Pull request merged**